frontend/src/setupTests.ts

```diff
@@ -5,8 +5,12 @@
 import '@testing-library/jest-dom/extend-expect';
 import 'jest-styled-components';
 
+// Unofficial adapter for React 17 for Enzyme.
+// See https://github.com/enzymejs/enzyme/issues/2429 for details
+import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
 import { configure } from 'enzyme';
-import Adapter from 'enzyme-adapter-react-16';
+
+configure({ adapter: new Adapter() });
```

Comment:
I have moved all Enzyme configuration to this central file that gets executed prior to running the tests. 
No need to configure Enzyme on each test individually